utiliti/src/main/java/de/gurkenlabs/utiliti/swing/dialogs/SpritesheetImportPanel.java

```diff
@@ -182,7 +180,7 @@ public Component getListCellRendererComponent(
     textField = new JTextField();
     ControlBehavior.apply(textField);
     textField.setColumns(10);
-    textField.addActionListener(e -> fileList.getSelectedValue().setName(textField.getText()));
+    fileList.addListSelectionListener(e -> textField.setText(fileList.getSelectedValue().getName()));
```

Comment:
I think the updates should be triggered as soon as the text field is updated, rather than when the selection in the file list changes. Try this:
```suggestion
   textField.getDocument().addDocumentListener(new DocumentListener() {
      @Override
      public void insertUpdate(DocumentEvent e) {
        updateSelectedFile();
      }

      @Override
      public void removeUpdate(DocumentEvent e) {
        updateSelectedFile();
      }

      @Override
      public void changedUpdate(DocumentEvent e) {
        updateSelectedFile();
      }
    });
```
Then later, define the `updateSelectedFile` method as follows:
```
  private void updateSelectedFile() {
    int selectedIndex = fileList.getSelectedIndex();
    if (selectedIndex != -1) {
      SpriteFileWrapper selectedFile = fileListModel.getElementAt(selectedIndex);
      selectedFile.setName(textField.getText());
      fileList.repaint();
    }
  }
```